interface/modules/custom_modules/oe-module-faxsms/ModuleManagerActionListener.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * Class to be called from Laminas Module Manager reporting management actions.
+ * Example is if module is disabled or unregistered ect.
+ *
+ * @package   OpenEMR Modules
+ * @link      https://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2024 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+/*
+ * Currently register isn't supported and support should be a part of install.
+ * If an error needs to be reported to user, return description of error.
+ * However, whatever action trapped here has already occurred in Manager.
+ * Catch any exceptions because chances are they will be overlooked in Laminas module.
+ * Report them in return value.
+*/
+
+class ModuleManagerActionListener
+{
+    // Prevent instantiation
```

Comment:
@bradymiller Do we have somewhere to put this file perhaps we have example module somewhere.